### PR TITLE
Support more XF86 Keycodes

### PR DESCRIPTION
**Pull request by Rom1deTroyes**
Signed-off-by: Romain Heller <Rom1deTroyes@users.noreply.github.com>

### CODE REVIEW

File: src/api/keymap/db/mediacontrols.js [hunk 1]
```diff
@@ -70,6 +70,22 @@ const MediaControlTable = {
         primary: "Vol-",
         verbose: "Volume down"
       }
+    },
+    {
+      code: 23663,
+      labels: {
+        top: "Media",
+        primary: "Brt+",
+        verbose: "Brithness up"
```
Comment (obra) [inline src/api/keymap/db/mediacontrols.js:79]: Brightness, rather than Brithness, throughout.

Comment (Rom1deTroyes) [reply src/api/keymap/db/mediacontrols.js]: Here comes the light :-)

**Review by algernon — APPROVED** [review]
Looks good to me!

**Comment by algernon** [comment]
@Rom1deTroyes Is there a reason why this is a draft? If there isn't, I'll merge it.

**Comment by Rom1deTroyes** [comment]
I'll plan to add more codes from the initial issue, so I made it as a draft, planning to add more commits latter...
I remove the `Draft` tag so it can be merged, and will pull some other requests later :-)

**Pull request merged**